src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -72,22 +72,60 @@ To set up the IAM role:
 
 When that's done, you can work on setting up your export rules using NerdGraph. For more on that, keep reading.
 
+
+## Set up an Azure Event Hub [#event-hub-setup]
+
+To set up streaming data export to Azure, you must first set up an Event Hub. We'll walk you through that procedure in the next three steps.
+
+Alternatively, you can follow the Azure guide [here](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create).
+
+### Step 1. Create an Event Hubs Namespace [#create-event-hubs-namespace]
+
+Create a dedicated Firehose to stream your New Relic data to:
```

Comment:
This should be changed from `Firehose` to `Event Hub`?

Reply:
🤦🏻 yes